src/metaswitch/clearwater/queue_manager/test/test_remove_from_queue_success.py

```diff
@@ -100,3 +100,4 @@ def test_remove_from_queue_success_not_front_of_queue(self):
         self.assertEqual(1, len(val.get("QUEUED")))
         self.assertEqual("10.0.0.2-node", val.get("QUEUED")[0]["ID"])
         self.assertEqual("PROCESSING", val.get("QUEUED")[0]["STATUS"])
+        alarms_patch.stop()
```

Comment:
I didn't mean to do that. Stopping the alarms patch causes crashes. This is an existing issue. I'm going to raise an issue to track it.